features/account/resetting_password.feature

```diff
@@ -40,3 +40,12 @@ Feature: Resetting a password
         And I confirm my new password as "newp@ssw0rd"
         And I reset it
         Then I should not be able to change my password again with the same token
+
+    @ui @email @api
+    Scenario: Trying to change my account password with an expired token I received
+        Given I have already received a resetting password email a while ago
+        When I follow link on my email to reset my password
+        And I specify my new password as "newp@ssw0rd"
+        And I confirm my new password as "newp@ssw0rd"
+        And I reset it
+        Then I should be notified that my password should not be reset
```

Comment:
I think this expectation could be renamed better. Perhaps `I should not be able to change my password`, similar to the expectation above? `my password should not be reset` just sounds awkward.